docs/dev-tools/zombienet/testing.md

```diff
@@ -3,7 +3,7 @@ title: Testing DSL
 description: Zombienet provides a Domain Specific Language (DSL) for writing tests. The DSL is designed to be human-readable and allows you to write tests using natural language expressions.
 ---
 
-# Testing DSL - ( 🚧 WIP )
+# Testing DSL
```

Comment:
So this is the only line I can comment on in this file, so I've got a few comments here:

1. In the list section in the intro, can you use this format?

- **On-chain storage** - description
- **Metrics** - description

-----------

2. PolkadotJs should be Polkadot.js

----------

3. filename - should be two words "file name"

----------

4. In the Running Tests section, the text there should be as follows:

To run the tests using the Native provider, you can use the Zombienet binary. The binary will read the test files and execute the tests defined in the DSL. The binary will output the results of the tests in the console.